README.md

```diff
@@ -1 +1,113 @@
-# ODAA
+# Datatidy
+
+Datatidy is a data wrangler application based on the Symfony 4 framework. It can take one or more datasources from public APIs, make some transformations and deliver the result to a datastore.
+
+## Getting started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Installing
+
+```bash
+$ docker-compose pull
+$ docker-compose up -d
+
+$ docker-compose exec phpfpm bin/install
```

Comment:
`bin/install`? Does this script exist?

Reply:
Heh well not yet - now using good old composer with 723cfd2